infertrade/utilities/operations.py

```diff
@@ -136,7 +146,6 @@ def research_over_price_minus_one(x: Union[np.ndarray, pd.Series], shift: int) -
 
 
 class PricePredictionFromSignalRegression(TransformerMixin, BaseEstimator):
-
     """Class for creating price predictions from signal values."""
```

Comment:
Spaces after Class definitions are [PEP8 compliant](https://stackoverflow.com/questions/17295244/blank-line-python-pep8-best-practice-in-class-definition), so I don't think we need to remove.